code/game/area/areas.dm

```diff
@@ -476,27 +481,13 @@
 	if((oldarea.has_gravity == 0) && (newarea.has_gravity == 1) && (L.m_intent == MOVE_INTENT_RUN)) // Being ready when you change areas gives you a chance to avoid falling all together.
 		thunk(L)
 
-	// Ambience goes down here -- make sure to list each area seperately for ease of adding things in later, thanks! Note: areas adjacent to each other should have the same sounds to prevent cutoff when possible.- LastyScratch
-	if(L && L.client && !L.client.ambience_playing && (L.client.prefs.sound & SOUND_BUZZ))	//split off the white noise from the rest of the ambience because of annoyance complaints - Kluys
+	//Ship ambience just loops if turned on.
+	if(L && L.client && !L.client.ambience_playing && (L.client.prefs.sound & SOUND_BUZZ))
 		L.client.ambience_playing = TRUE
```

Comment:
@Qustinnus 

You got rid of this, and I get why, but it causes a subtle cracking/popping noise  when you cross from one area to another---I suspect 99% of people won't notice it or just think it's an audio glitch---but I pick up on stuff like this and it drives me crazy.

Also, separate ship white-noise from ambient sounds on TG when? 😛

Reply:
i'll have a look, and yeah tbf good point i should make both work at once